doctr/__main__.py

```diff
@@ -122,6 +122,20 @@ def get_parser(config=None):
     deploy_parser = subcommand.add_parser('deploy', help="""Deploy the docs to GitHub from Travis.""")
     deploy_parser.set_defaults(func=deploy)
     deploy_parser_add_argument = make_parser_with_config_adder(deploy_parser, config)
+
+    _user_fork_parser = subcommand.add_parser('fork-deploy', help="""Deploy the docs to GitHub from Travis to the current user fork""")
```

Comment:
No, it could also be doctr-configure, but this command have to be run on user machine. 
I'm planning on having `doctr deploy` do automatically the right things is the USER_DEPLOY env variable is set.